src/modules/addons/lknhooknotification/src/Notifications/WhatsApp/Invoice6DaysLate/Invoice6DaysLateNotification.php

```diff
@@ -1,82 +1,103 @@
 <?php
+
 /**
- * Name: Fatura 6 dias atrasada
  * Code: Invoice6DaysLate
- * Platform: WhatsApp
- * Version: 1.0.0
- * Author: Link Nacional
  */
 
 namespace Lkn\HookNotification\Notifications\WhatsApp\Invoice6DaysLate;
 
 use DateTime;
 use Lkn\HookNotification\Config\Hooks;
+use Lkn\HookNotification\Config\ReportCategory;
 use Lkn\HookNotification\Domains\Platforms\WhatsApp\AbstractWhatsAppNotifcation;
+use Lkn\HookNotification\Helpers\Logger;
+use Lkn\HookNotification\Notifications\Chatwoot\WhatsAppPrivateNote\WhatsAppPrivateNoteNotification;
+use Throwable;
 
 final class Invoice6DaysLateNotification extends AbstractWhatsAppNotifcation
 {
     public string $notificationCode = 'Invoice6DaysLate';
-    public Hooks $hook = Hooks::DAILY_CRON_JOB;
+    public Hooks|array|null $hook = Hooks::DAILY_CRON_JOB;
 
-    public function run(): void
+    public function run(): bool
     {
+        $this->events = [];
+        $this->enableAutoReport = false;
+
+        $this->setReportCategory(ReportCategory::INVOICE);
+
         $invoices = localAPI('GetInvoices', [
-            'orderby' => 'invoicenumber',
-            'limitnum' => '250',
-            'status' => 'Overdue',
-            'orderby' => 'invoicenumber',
-            'order' => 'desc',
+            'limitnum' => 1000,
+            'status' => 'Overdue'
         ]);
 
         foreach ($invoices['invoices']['invoice'] as $invoice) {
             $givenDateTime = new DateTime($invoice['duedate']);
             $currentDateTime = new DateTime();
             $interval = $currentDateTime->diff($givenDateTime);
 
-            if ($interval->days === 6) {
-                if ($invoice['paymentmethod'] !== 'freeproducts') {
-                    if ($invoice['total'] !== '0.00') {
-                        $invoiceId = $invoice['id'];
-                        $clientId = $invoice['userid'];
+            if (
+                $interval->days !== 6
+                || $invoice['paymentmethod'] === 'freeproducts'
+                || $invoice['total'] === '0.00'
+            ) {
+                continue;
```

Comment:
@brunoferreiralkn Para simplificar não é melhor utillizar 'return' ou 'exit' em caso de invalidez?

Reply:
1. Observe that this notification runs on daily cron job and only for invoices 6 days late.
2. In this condition, if the invoice is not 6 days late, the notification keeps running to the next invoice. 
3. `return` or `exit` would stop the notification and not sending the message for the next possible 6 days late invoices.